lms/djangoapps/instructor/tasks_helper.py

```diff
@@ -56,17 +59,20 @@ def not_completed_query(course_id, query):
     """
     Specific db query for sections or problems that are not completed
     """
-    ids_in_course = CourseEnrollment.objects.filter(course_id=course_id,
-                                                    is_active=1,
-                                                    ).values_list(DatabaseFields.USER_ID)
+    #  Is this a repeat from not_open_query()?
+    #  Is it worth factoring out into its own method?
```

Comment:
I suppose we could split this out.

If you elect to do so, please do so in a separate commit. It can still be included in the same PR, but we'll want to isolate it so that the refactoring change is independent from general PEP8 cleanup.